src/SqlPersistence/Installer.cs

```diff
@@ -111,10 +141,19 @@ Task InstallSagas(string scriptDirectory, DbConnection connection, DbTransaction
 {string.Join(Environment.NewLine, scriptFiles)}");
         var sagaScripts = scriptFiles
             .Select(File.ReadAllText);
-        return connection.ExecuteTableCommand(
-            transaction: transaction,
-            scripts: sagaScripts,
-            tablePrefix: tablePrefix,
-            schema: schema);
+        if (sqlVariant == SqlVariant.Oracle)
+        {
+            return connection.ExecuteTableCommand(
+                transaction: transaction,
+                scripts: sagaScripts);
```

Comment:
It is not used on Oracle and documented here: https://github.com/Particular/docs.particular.net/blob/sql-oracle/nservicebus/sql-persistence/oracle-caveats.md#sagas.